package/com.unity.formats.usd/Tests/USD.NET/SceneTests.cs

```diff
@@ -119,5 +119,66 @@ public void WritePathToSceneFile_WithInvalidPath_ThrowsException(string path)
             Assert.Throws<System.Exception>(() => scene.Write(path, new SampleBase()));
             UnityEngine.TestTools.LogAssert.Expect(LogType.Exception, string.Format("ApplicationException: USD ERROR: Path must be an absolute path: <{0}>", path));
         }
+
+        [Test]
+        public static void StartEndTimeTest()
```

Comment:
Minor comment: As these are being moved could they also be given names that match the test name format followed by the other tests in this file?

/ I'm not sure whether there's much value in testing the assignment of a value, but maybe I'm missing something that can go wrong here :)

Reply:
I think this test was moved from StageTests.cs and I assume this file suits better the test.